amoro-mixed-format/amoro-mixed-format-spark/v3.2/amoro-mixed-format-spark-3.2/src/main/java/org/apache/amoro/spark/table/UnkeyedSparkTable.java

```diff
@@ -64,6 +56,14 @@ public String name() {
         + unkeyedTable.id().getTableName();
   }
 
+  @Override
+  public Map<String, String> properties() {
+    Map<String, String> properties = Maps.newHashMap();
+    properties.putAll(super.properties());
+    properties.put("provider", "arctic");
+    return properties;
+  }
```

Comment:
"provider" may should be "mixed-iceberg" or "mixed-hive"

Reply:
I have fixed it, PTAL.